diff --git a/node_modules/react-easy-panzoom/src/PanZoom.js b/node_modules/react-easy-panzoom/src/PanZoom.js
index 9987e46..6eff1af 100644
--- a/node_modules/react-easy-panzoom/src/PanZoom.js
+++ b/node_modules/react-easy-panzoom/src/PanZoom.js
@@ -249,6 +249,8 @@ class PanZoom extends React.Component<Props, State> {
     const offset = this.getOffset(e)
     this.zoomTo(offset.x, offset.y, scale)
     e.preventDefault()
+
+    return {dx: offset.x, dy: offset.y, scale: scale}
   }
 
   onKeyDown = (e: SyntheticKeyboardEvent<HTMLDivElement>) => {
